neo.UnitTests/UT_Scrypt.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Neo.Cryptography;
+using System;
+
+namespace Neo.UnitTests
+{
+    [TestClass]
+    public class UT_Scrypt
+    {
+        [TestMethod]
+        public void DeriveKeyTest()
+        {
+            int N = 16384, r = 8, p = 8;
+
+            var date = DateTime.UtcNow;
+            var derivedkey = SCrypt.DeriveKey(new byte[] { 0x01, 0x02, 0x03 }, new byte[] { 0x04, 0x05, 0x06 }, N, r, p, 64).ToHexString();
+            var elapsed = (DateTime.UtcNow - date);
+
+            Assert.AreEqual("2bb9c7bb9c392f0dd37821b76e42b01944902520f48d00946a51e72c960fba0a3c62a87d835c9df10a8ad66a04cdf02fbb10b9d7396c20959f28d6cb3ddfdffb", derivedkey);
+            Assert.IsTrue(elapsed.TotalSeconds > 1);
```

Comment:
This will fail when we use a super fast CPU.

Reply:
Then we should increase the scrypt values

Reply:
EHuaheauea, maybe just report the value is enough, Shargon, no?

Reply:
Is it intended to spend at least one second?

Reply:
Should be hard to bruteforce it

Reply:
And it also means 1more sec of UT...aejuaheuauhea

Reply:
I think tests should be deterministic.